package.json

```diff
@@ -46,6 +46,7 @@
     "test-watch": "karma start --auto-watch --no-single-run"
   },
   "dependencies": {
+    "connect-modrewrite": "^0.9.0",
```

Comment:
So do I need to do anything?  Maybe I can meet with you and you can check out my Git issue?